src/NUnitEngine/nunit.engine/Runners/DirectTestRunner.cs

```diff
@@ -104,32 +105,40 @@ protected override TestEngineResult LoadPackage()
             if (packages.Count == 0)
                 packages.Add(TestPackage);
 
-            var driverService = Services.GetService<IDriverService>();
+            var driverService = GetDriverService();
 
             foreach (var subPackage in packages)
             {
                 var testFile = subPackage.FullName;
 
+                string targetFramework = subPackage.GetSetting(InternalEnginePackageSettings.ImageTargetFrameworkName, (string)null);
+                bool skipNonTestAssemblies = subPackage.GetSetting(EnginePackageSettings.SkipNonTestAssemblies, false);
+
+#if !NETSTANDARD1_3
                 if (_assemblyResolver != null && !TestDomain.IsDefaultAppDomain()
                     && subPackage.GetSetting(InternalEnginePackageSettings.ImageRequiresDefaultAppDomainAssemblyResolver, false))
                 {
-                    // It's OK to do this in the loop because the Add method 
+                    // It's OK to do this in the loop because the Add method
                     // checks to see if the path is already present.
                     _assemblyResolver.AddPathFromFile(testFile);
                 }
 
-                string targetFramework = subPackage.GetSetting(InternalEnginePackageSettings.ImageTargetFrameworkName, (string)null);
-                bool skipNonTestAssemblies = subPackage.GetSetting(EnginePackageSettings.SkipNonTestAssemblies, false);
-                
                 IFrameworkDriver driver = driverService.GetDriver(TestDomain, testFile, targetFramework, skipNonTestAssemblies);
+#else
+                IFrameworkDriver driver = driverService.GetDriver(testFile, skipNonTestAssemblies);
+#endif
                 driver.ID = TestPackage.ID;
                 result.Add(LoadDriver(driver, testFile, subPackage));
                 _drivers.Add(driver);
             }
-
             return result;
         }
 
+        private IDriverService GetDriverService()
```

Comment:
I'm guessing there's been some complexity removed here, and it might make sense to re-inline this 1-line method.